controllers/applicationvolumereplication_controller.go

```diff
@@ -195,295 +191,471 @@ func (r *ApplicationVolumeReplicationReconciler) Reconcile(ctx context.Context,
-			(subscription.Spec.Placement != nil && subscription.Spec.Placement.Local != nil &&
-				*subscription.Spec.Placement.Local) {
-			a.log.Info("Skipping local subscription", "name", subscription.Name)
+	err := r.Client.Get(ctx, types.NamespacedName{
+		Name:      clonedPlRuleName,
+		Namespace: userPlRule.Namespace,
+	}, clonedPlRule)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			clonedPlRule, err = r.clonePlacementRule(ctx, avr, userPlRule, clonedPlRuleName)
+			if err != nil {
+				return nil, fmt.Errorf("failed to create cloned placementrule error: %w", err)
+			}
+		} else {
+			r.Log.Error(err, "Failed to get avr placementRule", "name", clonedPlRuleName)
 
-			continue
+			return nil, fmt.Errorf("failed to get placementrule error: %w", err)
 		}
+	}
 
-		placementDecision, needRequeue := a.processSubscription(&a.subscriptionList.Items[idx])
+	return clonedPlRule, nil
+}
 
-		if needRequeue {
-			a.log.Info("Requeue for subscription", "name", subscription.Name)
+func (r *ApplicationVolumeReplicationReconciler) clonePlacementRule(ctx context.Context,
+	avr *rmn.ApplicationVolumeReplication, userPlRule *plrv1.PlacementRule,
+	clonedPlRuleName string) (*plrv1.PlacementRule, error) {
+	r.Log.Info("Creating a clone placementRule from", "name", userPlRule.Name)
 
-			requeue = true
+	clonedPlRule := &plrv1.PlacementRule{}
 
-			continue
-		}
+	userPlRule.DeepCopyInto(clonedPlRule)
 
-		if a.instance.Status.Decisions == nil {
-			a.instance.Status.Decisions = make(rmn.SubscriptionPlacementDecisionMap)
-		}
+	clonedPlRule.Name = clonedPlRuleName
+	clonedPlRule.ResourceVersion = ""
+	clonedPlRule.Spec.SchedulerName = ""
 
-		if placementDecision != nil {
-			a.instance.Status.Decisions[subscription.Name] = placementDecision
-		}
+	err := r.addClusterPeersToPlacementRule(ctx, avr, clonedPlRule)
+	if err != nil {
+		r.Log.Error(err, "Failed to add cluster peers to cloned placementRule", "name", clonedPlRuleName)
+
+		return nil, err
 	}
 
-	a.log.Info("Returning Placement Decisions", "Total", len(a.instance.Status.Decisions))
+	err = r.Create(ctx, clonedPlRule)
+	if err != nil {
+		r.Log.Error(err, "failed to clone placement rule", "name", clonedPlRule.Name)
 
-	return requeue
+		return nil, errorswrapper.Wrap(err, "failed to coone PlacementRule")
+	}
+
+	return clonedPlRule, nil
 }
 
-func (a *AVRInstance) processSubscription(
-	subscription *subv1.Subscription) (*rmn.SubscriptionPlacementDecision, bool) {
-	a.log.Info("Processing subscription", "name", subscription.Name)
+func (r *ApplicationVolumeReplicationReconciler) addClusterPeersToPlacementRule(ctx context.Context,
+	avr *rmn.ApplicationVolumeReplication, plRule *plrv1.PlacementRule) error {
+	clPeersRef := avr.Spec.DRClusterPeersRef
+	clusterPeers := &rmn.DRClusterPeers{}
 
-	a.log.Info(fmt.Sprintf("processSubscription: current state '%s'",
-		a.instance.Status.LastKnownDRStates[subscription.Name]))
+	if clPeersRef.Namespace == "" {
+		clPeersRef.Namespace = avr.Namespace
```

Comment:
(future/note) Ultimately this should never be namespaced and cluster scoped instead.